Dynamo_Engine/Convert/ToDesignScript.cs

```diff
@@ -152,9 +152,18 @@ public static ADG.NurbsSurface ToDesignScript(this BHG.NurbsSurface surface)
             if (surface == null)
                 return null;
 
-            List<int> uvCount = surface.UVCount().Select(x => x -1).ToList(); // Align to Dynamo nurbs definition
+            List<int> uvCount = surface.UVCount().Select(x => x).ToList(); // Align to Dynamo nurbs definition
```

Comment:
Since `.Select(x => x).ToList(); // Align to Dynamo nurbs definition` is not doing anything anymore, can we remove it?
```c#
List<int> uvCount = surface.UVCount()
```